tests/multi-generic-sh

```diff
@@ -9,7 +9,7 @@ unifdef -M~ $opts $mfiles
 e=$?
 for f in $mfiles
 do
-	diff -u $f~ $f | sed '/^[+-]\{3\} /s/	.*//'
+	LANG=C diff -u $f~ $f | sed '/^[+-]\{3\} /s/	.*//'
```

Comment:
Good stuff, I had to do similar change to make this work for my fi_FI locale.

I would suggest to have export LC_ALL=C LANG=C just after export PATH line (line 3 in this scripts/runtests.sh)

Reply:
Sounds good! Updated.